src/services/items/service-api.ts

```diff
@@ -209,7 +209,15 @@ const plugin: FastifyPluginAsync = async (fastify) => {
 
         fastify.register(graaspCategoryPlugin);
 
-        fastify.register(graaspValidationPlugin);
+        fastify.register(graaspValidationPlugin, {
+          // replace this with real api from prod
+          classifierApi: 'http://172.17.0.1:8080/sync',
```

Comment:
Get this value from env.development, otherwise we won't be able to dynamically set it on aws.